internal/export/config.go

```diff
@@ -48,6 +48,12 @@ type Config struct {
 	TruncateWindow time.Duration `env:"TRUNCATE_WINDOW, default=1h"`
 	MinWindowAge   time.Duration `env:"MIN_WINDOW_AGE, default=2h"`
 	TTL            time.Duration `env:"CLEANUP_TTL, default=336h"`
+
+	// Debugging flags follow. These should not be enabled in production
+	// environments.
+
+	// DebugEndpointEnabled controls whether the debug endpoint is enabled.
+	DebugEndpoint bool `env:"DEBUG_ENDPOINT"`
```

Comment:
so - I think we want the debug endpoint on the exposure service not the export service.

in our current config, the export service is not accessible without IAM

Reply:
We may want to verify that both publish and export are using same truncate window. But if getting a /debug on publish/exposure service is lower hanging fruit that's totally fine.

Reply:
My plan was to do this on all services, just starting with one for now

Reply:
Looking more... `export` is guarded by auth already, so you'd need to have the right IAM permissions to invoke the /debug endpoint. `publish` is the only public-facing service where this makes sense, but the publisher won't have access to any of the export runtime configuration (unless we do something whacky like call the Cloud Run API, which seems like a bad ideA).